eng/common/pipelines/templates/steps/docs-metadata-release.yml

```diff
@@ -33,7 +33,7 @@ steps:
   displayName: Enable Long Paths if Necessary
   
 - pwsh: |
-    git clone https://github.com/${{ parameters.TargetDocRepoOwner }}/${{ parameters.TargetDocRepoName }} ${{ parameters.WorkingDirectory }}/repo
+    git clone https://$(azuresdk-github-pat)@github.com/${{ parameters.TargetDocRepoOwner }}/${{ parameters.TargetDocRepoName }} ${{ parameters.WorkingDirectory }}/repo
```

Comment:
Lets not do this here. It isn't needed for the clone only the remote which we are going to push.